src/store/getters/state-getters.ts

```diff
@@ -367,8 +372,55 @@ export const isIncorporationAgreementValid = (state: StateIF): boolean => {
   return getIncorporationAgreementStep(state).valid
 }
 
-/** Whether all the incorporation steps are valid. */
+/** Is true when the step is valid. */
+export const isDefineDissolutionValid = (state: StateIF): boolean => {
+  return isTypeCoop(state)
+    ? getDissolutionStatementStep(state).valid &&
+      getHasCertificateDestroyed(state)
+    : true
+}
```

Comment:
You will want to pull in `isCustodianValid`. 
The code just went in 👍 
That flag will apply to both Coops and Corps although the rules change slightly for each, the flag itself should be indicative of those.

The jist:
Coops: requires all the fields,
Corps: same as coop but `partyType` is not defaulted and needs to be selected and filled in (ie Person or Org).

Reply:
Added

Reply:
Yes, I see this has been added, but the logic could be cleaned up... in a future commit.

I am resolving this conversation but please keep track of this.